src/main/java/ch/njol/skript/bukkitutil/ItemUtils.java

```diff
@@ -36,27 +36,55 @@ public class ItemUtils {
 
 	/**
 	 * Gets damage/durability of an item, or 0 if it does not have damage.
-	 * @param stack Item.
+	 * @param itemStack Item.
 	 * @return Damage.
 	 */
-	public static int getDamage(ItemStack stack) {
-		ItemMeta meta = stack.getItemMeta();
-		if (meta instanceof Damageable)
+	public static int getDamage(ItemStack itemStack) {
+		ItemMeta meta = itemStack.getItemMeta();
+		if (meta instanceof Damageable) {
 			return ((Damageable) meta).getDamage();
+		}
```

Comment:
The [code conventions](https://github.com/SkriptLang/Skript/blob/master/code-conventions.md) permit no brackets if there's one statement in the conditional block, so this method didn't really need to change.

Reply:
Well you also have the option of if/else, but it's not really a good idea to change things if there's no reason tom since it messes up the Git history down the line. I'm not hugely bothered by it though.